.github/workflows/build-macos.yml

```diff
@@ -1,23 +1,44 @@
 name: Build macOS
 run-name: Build macOS
 on:
- push:
-   tags:
-     - "**"
-# on:
-#   push
+  push:
+    tags:
+      - "**"
 
 jobs:
   Build-macOS:
     runs-on: macos-latest
     steps:
-      # Use Python 3.12.4
       - uses: actions/checkout@v4
+      
       - uses: actions/setup-python@v5
         with:
           python-version: "3.12.4"
-      - run: pip install -r requirements.txt && pip install pyinstaller && python3 helper_scripts/package_mac.py && mkdir src/dist/Qtcord-macOS && mv src/dist/Qtcord-macOS.app src/dist/Qtcord-macOS
-      - uses: actions/upload-artifact@v4
+      
+      - name: Install dependencies and build
+        run: |
+          pip install -r requirements.txt
+          pip install pyinstaller
+          python3 helper_scripts/package_mac.py
+      
+      - name: Prepare release artifacts
+        run: |
+          mkdir -p src/dist/Qtcord-macOS
+          
+          # Create .dmg file
+          brew install create-dmg
+          create-dmg \
+            --volname "Qtcord" \
+            --window-pos 200 120 \
+            --window-size 800 400 \
+            --icon-size 100 \
+            --app-drop-link 600 185 \
+            Qtcord-${{github.ref_name}}-macOS-x86_64.dmg \
+            src/dist/Qtcord-macOS.app
+      
+      - name: Upload Release Assets
+        uses: softprops/action-gh-release@v1
         with:
-          name: "Qtcord-macOS"
-          path: "src/dist/Qtcord-macOS"
+          files: |
+            Qtcord-${{github.ref_name}}-macOS-x86_64.zip
```

Comment:
The build won't be x86, since macos-latest is running ARM64.

Reply:
If the zip file is being uploaded, where did it go?
![image](https://github.com/user-attachments/assets/18361616-7a00-4680-9707-aa9a6576f7ef)